db/schema.rb

```diff
@@ -873,9 +887,15 @@
   add_foreign_key "marathon_events", "marathons"
   add_foreign_key "marathons", "library_entries"
   add_foreign_key "media_follows", "users"
+  add_foreign_key "post_likes", "posts"
+  add_foreign_key "post_likes", "users"
   add_foreign_key "posts", "users"
   add_foreign_key "posts", "users", column: "target_user_id"
-  add_foreign_key "review_likes", "reviews"
+  add_foreign_key "quote_likes", "quotes"
+  add_foreign_key "quote_likes", "users"
+  add_foreign_key "quotes", "anime", column: "media_id"
```

Comment:
This foreign key will prevent the polymorphic association from being a Manga... kinda

Technically as long as the ID is also used by an Anime it'll pass the constraint, but that's not the kind of constraint you want